open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -323,22 +330,219 @@ DEFINE
+ ADDNAMESTREET             = CHARACTER(40)
+ ADDNAMESUBTYPE            = NUMBER
+ ADDNAMESUFFIX             = CHARACTER
+ ADDNAMETITLE              = CHARACTER
+ ADDNAMETYPE               = NUMBER
+ ADDNAMEUSPERSONFLAG       = NUMBER
+ ADDNAMEZIPCODE            = CHARACTER(10)
+ BNODATAFIELDCOUNT         = NUMBER
+ BNODATALINE               = CHARACTER
+ BNODATALINEFIELD          = CHARACTER ARRAY(19)
+ BNODATALINEFIELDMAX       = NUMBER
+ BNODATALINEINPUT          = CHARACTER
+ BNOPOS                    = NUMBER
+ COPYERRORTEXT             = CHARACTER
+ COPYINDEX                 = NUMBER
+ COPYNAMELISTLOOP          = NUMBER
+ COPYNAMESCHR              = CHARACTER
+ DUPLICATENAMEFOUND        = NUMBER
+ DUPNAMECOUNT              = NUMBER
+ EMAILHEADERPRINTED        = NUMBER
+ EMAILNAMECOUNT            = NUMBER
+ EMAILSENDERROR            = CHARACTER
+ LINKERRORTEXT             = CHARACTER
+ NAMECOPYLOOP              = NUMBER
+ NAMEFOUND                 = NUMBER
+ NAMEMERGEACTIVE           = NUMBER
+ NAMESTOCOPYLOC            = NUMBER ARRAY(19)
+ NAMESTOCOPYNAME           = CHARACTER ARRAY(19)
+ NAMESTOCOPYERROR          = CHARACTER ARRAY(19)
+ NAMESTOCOPYNAMETYPE       = NUMBER ARRAY(19)
+ NAMESTOCOPYCOUNT          = NUMBER
+ NAMESTOCOPYLOOP           = NUMBER
+ NAMETOCOPYLOC             = NUMBER
+ NAMESCOPIEDCOUNT          = NUMBER
+ NAMESCOPIEDNAMETYPE       = NUMBER ARRAY(99)
+ NAMESCOPIEDNAME           = CHARACTER ARRAY(99)
+ NAMESCOPIEDMAX            = 99
+ NEWNAMELOC                = NUMBER
+ PARSEARRAY                = CHARACTER ARRAY(999)
+ PARSEARRAYMAX             = NUMBER
+ PRIMARYSHORTNAME          = CHARACTER
+ SACCUSTAFFEMAILADDRESS    = CHARACTER
+ SACCUSTAFFEMAILSUBJECT    = CHARACTER
+ SACNAMECOPYLOANFLAG       = CHARACTER
+ SACNAMECOPYLOANTYPES      = CHARACTER
+ SACNAMECOPYSHAREFLAG      = CHARACTER
+ SACNAMECOPYSHARETYPES     = CHARACTER
+ SACTESTMODEACCOUNTLIST    = CHARACTER
+ SACTESTMODEFLAG           = CHARACTER
+ SLTYPELENGTH              = NUMBER
+ SLTYPESMAX                = NUMBER
+ TESTACCOUNT               = NUMBER
+ TESTACCTLIST              = CHARACTER ARRAY(10)
+ TESTACCTLISTCOUNT         = NUMBER
+ TESTACCTLISTMAX           = 10
+ TESTERRORCODE             = NUMBER
+ TESTMODE                  = NUMBER
+ TESTNAMECITY              = CHARACTER
+ TESTNAMEEXTRAADDRESS      = CHARACTER
+ TESTNAMEID                = CHARACTER
+ TESTNAMEIDTYPE            = NUMBER
+ TESTNAMELOCATOR           = NUMBER
+ TESTNAMESHORTNAME         = CHARACTER
+ TESTNAMESSN               = CHARACTER
+ TESTNAMESTATE             = CHARACTER
+ TESTNAMESTREET            = CHARACTER
+ TESTNAMEZIPCODE           = CHARACTER
+ TMPLOC                    = NUMBER
+ TMPNUM                    = NUMBER
+ TRUEFALSECHR              = CHARACTER ARRAY(1)
+ VALIDLOANNAMETYPES        = NUMBER ARRAY(99)
+ VALIDSHARENAMETYPES       = NUMBER ARRAY(99)
+ VALIDNAMETYPESCHR         = CHARACTER
+ VALIDNAMETYPES            = NUMBER ARRAY(99)
+
```

Comment:
I don't see VALIDNAMETYPESCHR and VALIDNAMETYPES used in the program. Are they for future use?

Note: I see this VALIDNAMETYPESCHR="ALL"  in code, but I don't see it used.